index.html

```diff
@@ -50,7 +47,276 @@ <h6>LET'S CONNECT</h6>
+             </button>
+             <button>
+                 css
+             </button>
+             <button>
+             javascript
+             </button>
+        </section>
+ 
+        <section class="big-button">
+                <button>
+                    See Project
+                </button>
+        </section>
  
+     </section>
+
+     <section class="works-four">
+       
+        <section class="image">
+            <img src="imagefour.png" alt="imageone">
+        </section>
+        
+        <section class="project-title">
+         <h2>Multi-Post <br> Stories</h2>
+         <section class="small-title">
+             <p>CANOPY</p>
+             <section class="list">
+                 <ul>
+                 <li>Back End Dev</li>
+                 <li>2015</li>
+                 </ul>
+             </section>
+       </section>
+
+ 
+        <section class="describe-project"> 
+           <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+        </section>
+ 
+        <section class="small-buttons">
+                    <button>  
+                        html
+                     </button>
+                     <button>
+                         css
+                     </button>
+                     <button>
+                     javascript
+                     </button>
+        </section>
+ 
+        <section class="big-button">
+                <button>
+                    See Project
+                </button>
+        </section>
+ 
+     </section>
+
+  <section class="aboutme">
+ 
+    <section class="mytitle">
+        <p>About <br> Myself</p>
+    </section>
+
+    <section class="describe-me">
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </section>
+
+    <section class="connect">
+          <p>LET'S CONNECT</p>
+          <ul>
+            <li><a href="https://twitter.com/"><img src="twitter.svg" alt="twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/"><img src="linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="//www.medium.com/"><img src="m.svg" alt="medium"></a></li>
+            <li><a href="https://github.com/"><img src="github.svg" alt="github"></a></li>
+            <li><a href="peace"><img src="peace.svg" alt="peace"></a></li>
+        </ul>
+
+    </section>
+    <section class="big-button">
+                <button>
+                    Get my Resume
+                </button>
+    </section>  
+
+   <section class="lang">
+      <section class="text">
+          <p>
+              Languages
+          </p>
+      </section>
+      <section class="dropdown">
+           <select name="dropdown" id="dropdown">      
+           </select>
+      </section>
+   </section> 
```

Comment:
- [ ] Your technologies  categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li